captum/attr/_core/neuron/neuron_guided_backprop_deconvnet.py

```diff
@@ -31,11 +42,11 @@ def __init__(self, model, layer, device_ids=None):
 
     def attribute(
         self,
-        inputs,
-        neuron_index,
-        additional_forward_args=None,
-        attribute_to_neuron_input=False,
-    ):
+        inputs: Union[Tensor, Tuple[Tensor]],
+        neuron_index: Union[int, Tuple[int]],
```

Comment:
This should be Tuple[int, ...], since it should take an arbitrary length tuple, not just a Tuple containing a single integer.